### DISCUSSION TITLE
How to affix the port in SSH command

### DISCUSSION THREAD

**Discussion by Svoch** [lima-vm/lima#810] [Q&A] [answered]
Every time I stop and restart my Lima instance, the port used for SSH changes. Is there a way to specify a single port to be assigned to an instance?

**Comment by AkihiroSuda** [comment, marked as answer]
The port number can be affixed in the `.ssh.localPort` property of the YAML


https://github.com/lima-vm/lima/blob/ae3e43ef9339b3e20c272f4d99865a9a784c9246/examples/default.yaml#L91-L96
